### ISSUE TITLE
[Race change][Passives] Orc pet dmg bonus bugged 

### ISSUE THREAD

**Issue by eri1997** [Atlantiss/NetherwingBugtracker#6656]
**Description**:  A character that race changed from troll to orc (qq) pet has less dmg compared to an orc character's pet (sandaya) that was orc from the start.
Both 0 buffs 0 gear on.
QQ:https://imgur.com/a/40DK8SY
Sandaya:https://imgur.com/a/b2zAmq4
**Current behaviour**: Passive 5% extra dmg on pets might not be applying after race change :(

**Expected behaviour**:  Passive working :)

**Server Revision**: 3259

**Comment by eri1997** [comment]
Did another test to see if it was a case of only old pets I had as a troll not getting the orc passive. Turns out that's not the case and even the new pets that I just tamed and leveled up today a wind serpent still gets bugged dmg values.
Mart's WS (orc character ):
https://imgur.com/a/fyFVJ83

Qq( orc that was racechanged from troll)
https://imgur.com/a/s4VbRCY

And here is a troll WS stats from another player:
https://imgur.com/a/xFt0ox2

As you can see WS dmg values are the same for the troll character and the orc that was racechanged from a troll. Marts dmg values on WS were higher as expected because of the orc passive.

So Tl;dr: Orc's 5% pet dmg increase passive not working on race changed characters at all.

**Comment by eri1997** [comment]
Did another test to see if it was a case of only old pets I had as a troll not getting the orc passive. Turns out that's not the case and even the new pets that I just tamed and leveled up today a wind serpent still gets bugged dmg values.
Mart's WS (orc character ):
https://imgur.com/a/fyFVJ83

Qq( orc that was racechanged from troll)
https://imgur.com/a/s4VbRCY

And here is a troll WS stats from another player:
https://imgur.com/a/xFt0ox2

As you can see WS dmg values are the same for the troll character and the orc that was racechanged from a troll. Marts dmg values on WS were higher as expected because of the orc passive.

So Tl;dr: Orc's 5% pet dmg increase passive not working on race changed characters at all.

**Comment by eri1997** [comment]
Hey @danbopes could you take a look at this quite a big dps loss . The passive name  is Command https://classic.wowhead.com/spell=21563/command .My ign is qq if you want to check anything

**Comment by eri1997** [comment]
Looking at wowhead looks like there is 2 spells for Command. The one I linked above didn't have hunter listed on classes. This one https://classic.wowhead.com/spell=20576/command only has hunter on it in case it helps.

**Comment by eri1997** [comment]
@Wolffenstein @danbopes  Went and checked with the help of Easy. So the problem is you put the wrong Command ID on hunters that have race changed to orc. For some reason there are 2 Command spells and 1 of them isn't for hunters.
Here is my race changed command spell:
https://imgur.com/a/QNhUYqv
Here is Mart Orc Hunter command spell the correct one:
https://imgur.com/a/u5XMw4y

The correct spell id is 21563.

**Comment by danbopes** [comment]
Was this in the ptr? This was already fixed on live and your stats should
be correct.

**Comment by eri1997** [comment]
Ok so it looks like it got fixed on live didn't see this closed or anything on the changelog so didn't check if anything had changed live but yes the fix is working pet dmg values seem correct right now. You can close this thank you very much for fixing it
